quadrat/block-template-parts/404.html

```diff
@@ -2,7 +2,7 @@
 <div class="wp-block-group">
 
 <!-- wp:heading {"textAlign":"center","level":1,"fontSize":"large"} -->
-<h1 class="has-text-align-center has-large-font-size">Oops! That page can’t be found.</h1>
+<h1 class="has-text-align-center has-large-font-size">Quadrat Oops! That page can’t be found.</h1>
```

Comment:
I think this change comes from testing template loading, right? cause I've done the same :D

Reply:
Yeah :)